pkg/ccl/importccl/sst_writer_proc.go

```diff
@@ -214,7 +214,13 @@ func (sp *sstWriter) Run(ctx context.Context) {
 						// throughput.
 						log.Errorf(ctx, "failed to scatter span %s: %s", roachpb.PrettyPrintKey(nil, end), pErr)
 					}
-					if err := bulk.AddSSTable(ctx, sp.db, sst.span.Key, sst.span.EndKey, sst.data, false /* disallowShadowing */); err != nil {
+					// TODO(adityamaru): There is an observed 1.1% overhead of always
+					// disallowing shadow keys in AddSSTable i.e. when no collisions are
+					// found. Since IMPORT TABLE ingests data into a new key space there
+					// can never be any shadow keys, and this check could be turned off.
```

Comment:
Direct-ingest import has no sort so we _could_ collide with our own keys even in new table import.

Reply:
ugh, never mind, I got confused.